rest_framework/parsers.py

```diff
@@ -183,7 +183,7 @@ def parse(self, stream, media_type=None, parser_context=None):
 
         for index, handler in enumerate(upload_handlers):
             file_obj = handler.file_complete(counters[index])
-            if file_obj:
```

Comment:
Have we got any idea why the `file_obj` was returning false?
Your example seemed to make it clear that it wasn't simply an empty file, so it's a bit unclear to me *exactly* what cases this changes the behavior for, and why?